eox_hooks/utils.py

```diff
@@ -0,0 +1,50 @@
+"""
+Utils functions for actions.
+"""
+import collections
+
+from django.conf import settings
+
+
+def flatten_dict(d, parent_key='', sep='_'):
```

Comment:
Could you add a unit test for this function? @magajh

Reply:
Done @morenol